common/display/displayqueue.cpp

```diff
@@ -563,7 +569,7 @@ bool DisplayQueue::QueueUpdate(std::vector<HwcLayer*>& source_layers,
     bool needs_plane_validation = false;
     GetCachedLayers(layers, remove_index, &current_composition_planes,
                     &render_layers, &can_ignore_commit, &needs_plane_validation,
-                    &validate_layers);
+                    &validate_layers, force_media_composition);
```

Comment:
This should be all managed at https://github.com/intel/IA-Hardware-Composer/blob/master/common/display/displayqueue.cpp#L591 

If this is not doing what it's supposed do, we should fix it their.